src/hotspot/share/compiler/compileLog.hpp

```diff
@@ -69,7 +70,9 @@ class CompileLog : public xmlStream {
   // or reset, context string will be silently ignored
   stringStream* context()                        { return &_context; }
   void    clear_context()                        { context()->reset(); }
-  void      set_context(const char* format, ...) ATTRIBUTE_PRINTF(2, 3);
+
+  ATTRIBUTE_PRINTF(2, 3)
+  void      set_context(const char* format, ...);
```

Comment:
Whitespace between return type and function name is pretty pointless here.  Also above.